synfig-studio/src/gui/dialogs/dialog_setup.cpp

```diff
@@ -413,6 +415,16 @@ Dialog_Setup::create_editing_page(PageInfo pi)
 	toggle_resize_imported_images.set_tooltip_text(_("When you import images, check this option if you want they fit the Canvas size."));
 	toggle_resize_imported_images.set_halign(Gtk::ALIGN_START);
 	toggle_resize_imported_images.set_hexpand(false);
+		
+	// Editing Animation section
+	attach_label_section(pi.grid, _("Animation"), ++row);
+
+        // Editing - Show animation thumbnail preview
+        attach_label(pi.grid, _("Thumbnail preview"), ++row);
+        pi.grid->attach(toggle_animation_thumbnail_preview, 1, row, 1, 1);
+        toggle_animation_thumbnail_preview.set_tooltip_text(_("Turn on/off animation thumbnail preview when you hover your mouse over the timetrack panel."));
+        toggle_animation_thumbnail_preview.set_halign(Gtk::ALIGN_START);
+        toggle_animation_thumbnail_preview.set_hexpand(false);
```

Comment:
@Keyikedalube Can you please fix indentation here? ^__^

Reply:
Done ;)